### PR TITLE
add some instructions on data transfer RAP-VM

### PR DESCRIPTION
**Pull request by R-Walmsley**

### CODE REVIEW

**Comment by R-Walmsley** [comment]
@gmertes I have roughly tested these. We can then point the students to them.

File: _posts/2022-11-30-prac5-6-Association-Analysis-in-UK-Biobank.md [hunk 1]
```diff
@@ -60,6 +60,54 @@ Alternatively, if you already have things in permanent storage (e.g. having prev
...
+
+### Steps
+
+1. Login to the VM and complete all steps below on the VM.
+
+2. Load a pip environment (to allow you control over package installation):
+```bash
+python3 -m venv env_for_dx
+source env_for_dx/bin/activate
+```
```
Comment (gmertes) [inline _posts/2022-11-30-prac5-6-Association-Analysis-in-UK-Biobank.md]: I would change this to their existing anaconda environments, I tested it with the template environment and there don't seem to be dependency conflicts. So you could change the install instructions to just pip install in their conda environment:

```
conda activate wearables_workshop
pip3 install dxpy
```

**Review by gmertes — CHANGES_REQUESTED** [review]
Thanks Rosemary, this looks great! I would just swap out venv for their conda environments, since they'll have that active most, if not all, of the time anyway. Then the dx tool is always easily available.

File: _posts/2022-11-30-prac5-6-Association-Analysis-in-UK-Biobank.md (hunk 1 above)
Comment (R-Walmsley) [reply _posts/2022-11-30-prac5-6-Association-Analysis-in-UK-Biobank.md]: Thanks, change made. Good spot as I had no idea they had these set up!

**Review by gmertes — APPROVED** [review]

**Pull request merged**